aspnetcore/grpc/comparison.md

```diff
@@ -7,11 +7,11 @@ ms.author: jamesnk
 ms.date: 03/26/2019
 uid: grpc/comparison
 ---
-# Comparing gRPC and HTTP APIs
+# Comparing gRPC services with HTTP APIs
 
 By [James Newton-King](https://twitter.com/jamesnk)
 
-This article provides a comparison between [gRPC](https://grpc.io/docs/guides/) and HTTP APIs, and recommends scenarios for using gRPC over other technologies.
+This article provides a comparison between [gRPC services](https://grpc.io/docs/guides/) and HTTP APIs, and recommends scenarios for using gRPC over other technologies.
```

Comment:
To avoid that comma splice (before the last "and") and avoid the awkward and...and structure, I recommend using the list format for this (we do it quite frequently to break up long sentences for machine translation anyway) ...

> This article provides:
>
> \* A comparison between \[gRPC services](https://grpc.io/docs/guides/) and HTTP APIs.
> \* Recommended scenarios for using gRPC over other technologies.
```

Reply:
I'd like a more conversational paragraph to introduce the article. I've made some changes.

Reply:
I generally agree with your sentiment.